app/src/behaviors/behavior_hold_tap.c

```diff
@@ -36,12 +36,30 @@ enum flavor {
     ZMK_BHV_HOLD_TAP_FLAVOR_TAP_PREFERRED = 2,
 };
 
+enum status {
+    STATUS_UNDECIDED = 0,
+    STATUS_TAP = 1,
+    STATUS_HOLD_INTERRUPT = 2,
+    STATUS_HOLD_TIMER = 4,
+};
+
+#define STATUS_HOLD (STATUS_HOLD_INTERRUPT | STATUS_HOLD_TIMER)
```

Comment:
Since this appears to only be used as a bit mask and isn't a valid value, maybe rename to `STATUS_HOLD_MASK`?